generator/templates/schemapolymorphic.gotmpl

```diff
@@ -8,6 +8,8 @@
         the definition of the base type has a Validate() func.
     */}}
         runtime.Validatable
+        // WIP: add to interface in runtime pkg
+        ContextValidate(context.Context, strfmt.Registry) error
```

Comment:
This should go in runtime pkg

Reply:
Yeah you're right it can go in there. Most of them are here: https://github.com/go-openapi/runtime/blob/master/interfaces.go

Reply:
https://github.com/go-openapi/runtime/pull/185 added PR here.

Reply:
fixed.